### ISSUE TITLE
PHP warning for makeitsecure_ssl

### ISSUE THREAD

**Issue by ryanmarkel** [ryanmarkel/https-all-the-things#1]
This was reported in https://github.com/ryanmarkel/ryanmarkel-v2/issues/6 by @simonwheatley: 

```
( ! ) Warning: call_user_func_array() expects parameter 1 to be a valid callback, class 'makeitsecure_ssl' does not have a method 'action_init' in /srv/www/wordpress-default/wp-includes/plugin.php on line 496
```

**Comment by ryanmarkel** [comment]
I'm posting this here because I have to admit that I don't understand what's wrong with it. It's not my code and I have a really hard time understanding object-oriented PHP. If there's anyone who would like to submit a fix and also explain what's going on and why this is happening, I would love to read it.

**Comment by ryanmarkel** [comment]
I think I broke this in d6d4b9aef65ead1ef8663436900fda8c892f1464.

EDIT: In looking at it I don't know why this didn't cripple the whole thing.

**Comment by cohix** [comment]
Can you post the line in plugin.php that calls the makeitsecure_ssl class constructor?

**Comment by ryanmarkel** [comment]
> Can you post the line in plugin.php that calls the makeitsecure_ssl class constructor?

It's just part of WP core.

**Comment by cohix** [comment]
Mmk well you'll probably have to re-add that function and re-work it so it doesn't cause 301 loops somehow ¯_(ツ)_/¯

**Comment by ryanmarkel** [comment]
I think I understand now why people have things in their code that cause PHP warnings and then just ignore them and never fix them.

**Comment by cohix** [comment]
I mean, the PHP Interpreter has very sub-optimal errors. They're often very confusing and Google-worthy.

**Comment by ryanmarkel** [comment]
This was solved in #3.
